tests/views/test_collection_exercise.py

```diff
@@ -18,6 +19,9 @@
 class TestCollectionExercise(unittest.TestCase):
 
     def setUp(self):
+        app_config = TestingConfig()
```

Comment:
If this method of setting things up to use TestingConfig works it's definitely the way to go. We should probably do this in all our other test files as well

Reply:
I'm pretty sure it works, but looking at the config documentation on flask, they have something like this which we could look at...   `app.config.from_object('configmodule.ProductionConfig')`

Reply:
The above is functionally the same as passing a string to a module that contains a config class.